bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/dto/clip2/Dimming.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hue.internal.dto.clip2;
+
+import java.util.Objects;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.hue.internal.exceptions.DTOPresentButEmptyException;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * DTO for dimming brightness of a light.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@NonNullByDefault
+public class Dimming {
+    private @Nullable Double brightness;
+    private @Nullable @SerializedName("min_dim_level") Double minimumDimmingLevel;
+
+    public static final double DEFAULT_MINIMUM_DIMMIMG_LEVEL = 0.5f;
+
+    /**
+     * @throws DTOPresentButEmptyException to indicate that the DTO is present but empty.
+     */
+    public double getBrightness() throws DTOPresentButEmptyException {
+        Double brightness = this.brightness;
+        if (Objects.nonNull(brightness)) {
+            return brightness;
+        }
+        throw new DTOPresentButEmptyException("'dimming' DTO is present but empty");
+    }
+
+    public @Nullable Double getMinimumDimmingLevel() {
+        return minimumDimmingLevel;
+    }
+
+    public Dimming setBrightness(double brightness) {
+        this.brightness = brightness;
+        return this;
+    }
+
+    public Dimming setMinimumDimmingLevel(Double minimumDimmingLevel) {
+        this.minimumDimmingLevel = minimumDimmingLevel;
+        return this;
+    }
+
+    public @Nullable String toPropertyValue() {
+        Double minimumDimmingLevel = this.minimumDimmingLevel;
+        if (Objects.nonNull(minimumDimmingLevel)) {
+            return String.format("%.1f %% .. 100 %%", minimumDimmingLevel.doubleValue());
```

Comment:
Is " .. " a standard notation for defining a range?

Reply:
I don't think there is anything so formal in English syntax for this. Some people show it as `min - max`, but I prefer `min .. max` in order to avoid confusion between negative sign '-' and dash sign '-'. However I can change this to anything you want.

Reply:
PS the origin is probably the Pascal language where set constants are declared as `[1, 2, 5, 9]` or `[1 .. 9]` or even `[1 .. 3, 7 .. 9]` -- see https://en.wikibooks.org/wiki/Pascal_Programming/Sets#Set_literals -- I suppose I learned it there.